### PR TITLE
Adds link[@rel='me'] support to site config, populate with Fosstodon link

### PR DESCRIPTION
**Pull request by colindean**
- Adds a link header validating Fosstodon profile
- Extract link[@rel="me"] to a site variable

### CODE REVIEW

**Review by woodruffw — APPROVED** [review]
LGTM modulo final decision on the Mastodon instance!

**Review by woodruffw — APPROVED** [review]

**Review by MikeMcQuaid — APPROVED** [review]

**Pull request merged**
